protobufs/livekit_sip.proto

```diff
@@ -197,6 +205,7 @@ message SIPInboundTrunkInfo {
   google.protobuf.Duration max_call_duration = 12;
 
   bool krisp_enabled = 13;
+  SIPMediaEncryption media_encryption = 14;
```

Comment:
Similarly to krisp, this allows to set the encryption setting for each call (participant) for outbound, but makes it a trunk global setting for inbound. Is there any reason not to put these 2 settings on the dispatch? This seems it would be more consistent.